modyn/supervisor/supervisor.py

```diff
@@ -102,15 +117,21 @@ def validate_pipeline_config_schema(self) -> bool:
 
         return True
 
-    @staticmethod
-    def _validate_evaluation_options(evaluation_config: dict) -> bool:
+    def _validate_evaluation_options(self, evaluation_config: dict) -> bool:
         is_valid = True
 
         dataset_ids = [dataset["dataset_id"] for dataset in evaluation_config["datasets"]]
         if len(set(dataset_ids)) < len(dataset_ids):
             logger.error("Dataset ids must be unique in evaluation")
             is_valid = False
 
+        if "result_writers" in evaluation_config:
+            writer_names = set(evaluation_config["result_writers"])
+            supported_names = {writer.get_name() for writer in self.supported_evaluation_result_writers}
```

Comment:
We could also handle this similary to some other cases within Modyn where we have an enum that maps to class names. Would that make sense? not sure what is the best approach

Reply:
Sure! I did some changes in order to solve this differently (see also other comment).